tests/scripts/test_move_session_workspace.py

```
@@ -0,0 +1,736 @@
+
+def test_cli_runs_as_script_without_name_error():
+    """Regression: running the module as a script must not NameError because a
+    function (plan_moves) is defined after the __main__ guard. --from==--to hits
+    the same-workspace guard before any DB query, so this needs no database."""
+    import os
+    import subprocess
+    import sys
+
+    repo = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+    script = os.path.join(repo, "scripts", "move_session_workspace.py")
+    result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)